spec/models/event_spec.rb

```diff
@@ -1,8 +1,136 @@
 require 'spec_helper'
 
 describe Event do
+  let(:event) { create(:event) }
+  let(:user) { create(:user) }
 
-  describe 'abstract_word_count' do
+  describe 'validations' do
+    it 'has a valid factory' do
+      expect(build(:event)).to be_valid
+    end
+
+    it 'is not valid without a title' do
```

Comment:
Furthermore, the ``should`` syntax is deprecated!
http://betterspecs.org/#expect